### DISCUSSION TITLE
Please enter a correct username and password.

### DISCUSSION THREAD

**Discussion by alemsas** [netbox-community/netbox-docker#1213] [Getting Help]
I upgraded my NetBox instance to version 3.6.5 using Docker, but now when I try to log in with my username and password, I receive the following error:

"Please enter a correct username and password. Note that both fields may be case-sensitive."

When I check the database, I can see that the user information is present in the user table. Additionally, all containers including NetBox, Redis, and PostgreSQL are up and running without any visible errors.

Could someone please help me resolve this login issue?

**Comment by alemsas** [comment]
i solve above!
now i want to import my database into my container and i get!
`pg_restore: error: unsupported version (1.15) in file header`

**Comment by natashachaika** [comment]
> I upgraded my NetBox instance to version 3.6.5 using Docker, but now when I try to log in with my username and password, I receive the following error:
> 
> "Please enter a correct username and password. Note that both fields may be case-sensitive."
> 
> When I check the database, I can see that the user information is present in the user table. Additionally, all containers including NetBox, Redis, and PostgreSQL are up and running without any visible errors.
> 
> Could someone please help me resolve this login issue?


can you please advise how did you fix it? Having similar issue
